examples/complete/variables.tf

```diff
@@ -32,3 +32,9 @@ variable "resource_tags" {
   description = "Optional list of tags to be added to created resources"
   default     = []
 }
+
+variable "en_region" {
```

Comment:
why do we need a separate region for event notifications? Not sure this will work as the var.region is used in the provider config

Reply:
<img width="512" alt="Screenshot 2024-01-10 at 4 37 38 PM" src="https://github.com/terraform-ibm-modules/terraform-ibm-secrets-manager/assets/122446118/1902af5d-61fc-4d72-952a-da610c13c581">

Event notifications are available in the above regions, but the secret manager can be placed in any region. we was facing issue due to the provider config using var.region, which was fixed in this PR. https://github.com/terraform-ibm-modules/terraform-ibm-event-notifications/pull/120

Reply:
ok I guess this is fine then